BusinessServices/PlayerManager.cs

```diff
@@ -55,7 +64,7 @@ public Player AddPlayer(PlayerHandshake info)
             //and a couple more in the same cell as the player
             //RockGenerator.CreateRandomRocks(newPlayer.CurrentCell);
             Players.Add(newPlayer);
-            Players.Add(new Player(null, WorldState.Instance.GetRandomGridCell(),new Vector2(50,50), _playerColors[info.Name.Min() % _playerColors.Length]));
+            Players.Add(new Player(null, WorldState.Instance.GetRandomGridCell(),new Vector2(50,50), _playerColors[new Random().Next(_playerColors.Length)]));
```

Comment:
at least create a private member for the Random object for the Manager haha

Reply:
lolol, but jeff, don't you enjoy the overhead of the random class being instanced every time?

Also, why'd I change it?.... Oh, when I make the fake player, they don't have a randomized name like regular players, that's why. 

Other than that, I thought your method was more clever

In fact, this makes for an interesting mechanic: Player's color/appearance, being generated from the name always. That way their character would always look the same as long as they used the same name. Gives it a monster rancher feel